packages/utilities/psammead-storybook-helpers/src/index.stories.jsx

```diff
@@ -22,7 +22,7 @@ storiesOf('Utilities|Input Provider', module)
         },
         { name: 'second slot' },
       ],
-      ([first, second], script, dir) => (
+      ({ inputs: [first, second], script, dir }) => (
         <ul>
           <li>{first}</li>
           <li>{second}</li>
```

Comment:
You could add an `<li>` showing off that you get the service name back from the input provider here.

There's also a few references in this file to 'language' or 'English' - they could be updated to 'service' and 'News'